model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -173,10 +173,7 @@ class OrtResultTest : WordSpec({
 
             val openIssues = ortResult.getOpenIssues(Severity.WARNING)
 
-            openIssues should haveSize(1)
-            with(openIssues.first()) {
-                message shouldBe "Non-resolved issue"
-            }
+            openIssues.map { it.message }.shouldContainExactly("Non-resolved issue")
```

Comment:
Nit: Closer to the original semantics would be `shouldHaveSingleElement` (which also does not take a `vararg` and thus has an `infix` version).

Reply:
Why does it have to be "close" to the previous code?
I don't see why it is closer. The semantics is the same, isn't it.

Reply:
`shouldHaveSingleElement "Non-resolved issue"` has the exact same semantics as `.shouldContainExactly("Non-resolved issue")`, doesn't it?

Changing it anyway, as `shouldHaveSingleElement` is a slight abbrevation.

Reply:
> `shouldHaveSingleElement "Non-resolved issue"` has the exact same semantics as `.shouldContainExactly("Non-resolved issue")`, doesn't it?

`shouldContainExactly` *alone* does not have the same semantics as `shouldHaveSingleElement`, you also need to take the amount of parameters into account to match semantics. In other words, `shouldHaveSingleElement` better expresses that you're *never* expecting there to be more than a single element.